examples/flavor/swarm/flavor.go

```diff
@@ -228,6 +257,10 @@ func (s *baseFlavor) prepare(role string, flavorProperties *types.Any, instanceS
 			log.Warningln("Cannot prepare:", err)
 		}
 
+		if dockerClient != nil {
+			dockerClient.(io.Closer).Close()
```

Comment:
Hi @YujiOshima `io.Closer` is just an interface. the `Close` function that will be called is the one defined on `client.Client`. Anyways, I have to update this PR because I forgot things on the unit tests side.